Source/ASTableNode.mm

```diff
@@ -80,7 +80,7 @@ - (instancetype)initWithStyle:(UITableViewStyle)style
     [self setViewBlock:^{
       // Variable will be unused if event logging is off.
       __unused __typeof__(self) strongSelf = weakSelf;
```

Comment:
Is the strongSelf necessary? If the viewBlock is running, self shouldn't be nil, right?